z3c/dependencychecker/db.py

```diff
@@ -43,3 +48,186 @@ def _filter_duplicates(self, imports):
         all_imports = set([dotted_name for dotted_name in imports])
         filtered = all_imports - self._requirements
         return filtered
+
+    def add_imports(self, imports):
+        for single_import in imports:
+            self.imports_used.append(single_import)
+
+    def get_missing_imports(self):
+        filters = (
+            self._filter_known_packages,
+            self._filter_testing_imports,
+            self._filter_own_package,
+            self._filter_requirements,
+            self._filter_python_standard_library,
+        )
+        filtered = self._process_pipeline(self.imports_used, filters)
```

Comment:
Perhaps rename "filtered" to "missing"? You "filter out" stdlib imports, known packages, etcetera. So what's left isn't "filtered", but "remaining". (Or in this case, "missing", I guess).

Reply:
Done